app/screens/send-bitcoin-screen/destination-information.tsx

```diff
@@ -6,21 +6,22 @@ import { useAppConfig } from "@app/hooks"
 import React from "react"
 import { Text, View } from "react-native"
 import EStyleSheet from "react-native-extended-stylesheet"
-import { SendBitcoinDestinationState } from "./send-bitcoin-reducer"
+import { DestinationState, SendBitcoinDestinationState } from "./send-bitcoin-reducer"
 import { IntraledgerPaymentDestination } from "@galoymoney/client/dist/parsing-v2"
+import { InvalidDestinationReason } from "./payment-destination/index.types"
 
 const toLnAddress = (handle: string, lnDomain: string) => {
   return `${handle}@${lnDomain}`
 }
 
 const destinationStateToInformation = (
-  destinationState: SendBitcoinDestinationState,
+  sendBitcoinReducerState: SendBitcoinDestinationState,
```

Comment:
I think we should also keep, for consistency, the pattern we have in the backend of using object are argument, instead of positional argument. 

we can do those changes in another PR

Reply:
This can be addressed with issue #1089